test/evergreen.yml

```diff
@@ -128,6 +128,14 @@ tasks:
           configure_env_vars: CC=/opt/mongodbtoolchain/v3/bin/clang CXX=/opt/mongodbtoolchain/v3/bin/clang++ ASAN_OPTIONS=detect_leaks=1:abort_on_error=1:disable_coredump=0 ASAN_SYMBOLIZER_PATH=/opt/mongodbtoolchain/v3/bin/llvm-symbolizer CFLAGS=-fsanitize=address
           posix_configure_flags: --enable-silent-rules --enable-strict --enable-diagnostic --disable-static
 
+  - name: compile-msan
+    commands:
+      - func: "get project"
+      - func: "compile wiredtiger"
+        vars:
+          configure_env_vars: CC=/opt/mongodbtoolchain/v3/bin/clang CXX=/opt/mongodbtoolchain/v3/bin/clang++ MSAN_OPTIONS=abort_on_error=1:disable_coredump=0 MSAN_SYMBOLIZER_PATH=/opt/mongodbtoolchain/v3/bin/llvm-symbolizer CFLAGS=-fsanitize=memory
```

Comment:
`CFLAGS=-fsanitize=memory` looks weird. Guess it's working fine as the new evergreen tasks ran successfully.

Reply:
I think it's fine. That's how I usually compile with ASan and UBSan. Also, the ASan test in this file looks similar.